src/content/docs/implementation/setup/content-delivery-network.mdx

```diff
@@ -0,0 +1,605 @@
+
+```txt
+if (req.http.x-mesh == "true") {
+    # There's a bug in API MESH that combines multiple set-cookie headers
+    # Let's remove these cookies from the response
+    declare local var.ignored BOOL;
+    set var.ignored = setcookie.delete_by_name(beresp, "private_content_version");
+    set var.ignored = setcookie.delete_by_name(beresp, "form_key");
+    set var.ignored = setcookie.delete_by_name(beresp, "authentication_flag");
+    set var.ignored = setcookie.delete_by_name(beresp, "dataservices_customer_id");
+    set var.ignored = setcookie.delete_by_name(beresp, "dataservices_customer_group");
+    set var.ignored = setcookie.delete_by_name(beresp, "dataservices_cart_id");
+    return (pass);
+}
+```
+
+:::note
+`x-mesh` header is just an example. It would be added in API Mesh to use as determination in routing of VCL (for example, `x-commerce-bypass-fastly-cache`).
+:::
+
+## Commerce configuration
+
+If you are using the VCL to do the APEX redirect (`aemshop.net` → `www.aemshop.net` with 801 error), you should do two things in the Adobe Commerce backend:
+
+1. Disable `auto-redirect`.
+1. Set the `base_url` and `secure_base_url` settings to your domain, including `www`.
+
+Follow these steps:
+
+<Steps>
+
+1. Log in to the Adobe Commerce Admin.
+
+1. Click **Stores** > _Settings_ > **Configuration** > **General** > **Web**.
+
+1. Expand the **URL options** section.
+
+1. Set **Auto-redirect to Base URL** to **No**.
+
+1. Expand the **Base URLs** section.
+
+1. Set the **Base URL** and **Secure Base URL** to your domain, including `www`.
+
+   For example: `https://www.aemshop.net/`
+
+1. Expand the **Base URLs (Secure)** section.
+
+1. Set the **Secure Base URL** to your domain, including `www`.
+
+   For example: `https://www.aemshop.net/`
+
+1. Click **Save Config**.
+
+</Steps>
+
+## Edge Delivery Service configuration
+
+To obtain a purge API token for Fastly, you must contact Adobe Commerce Customer Support.
+
+Then, follow the instructions in [Setup push invalidation for Fastly](https://www.aem.live/docs/byo-cdn-fastly-setup#setup-push-invalidation-for-fastly). The remaining configuration described on this page is already taken care of by applying the VCL snippets above.
+
+## Validation
+
+To validate your CDN setup, use `curl` requests to check expected responses from the paths you have configured.
+
+### Content encoding, surrogate key, and cache
+
+Validate that surrogate key, cache hits, and content encoding are working as expected by requesting an Edge Delivery Services-served asset from your Commerce domain. Ensure you are checking a warm cache by making the request at least twice. The following examples are to a warmed cache.
+
+Here is an example of validation against an Adobe staging environment:
+
+```bash
+curl -sI -H 'Fastly-Debug: 1' https://www.aemshop.net/scripts/aem.js | grep 'x-cache\|surrogate-key\|cache-control'
+content-encoding: gzip
```

Comment:
I think the response begins here.

This comment is applicable for all the curl commands in this section. The commands and responses are in the same bash gates, without even a blank line to separate them.